src/number_theory/divisors.lean

```diff
@@ -233,6 +301,54 @@ begin
   apply h.ne_one.symm,
 end
 
+lemma proper_divisors_eq_singleton_one_iff_prime :
+  n.proper_divisors = {1} ↔ n.prime :=
```

Comment:
I guess `nat.prime.proper_divisors` (the mpr direction) could be extracted. That way it will be a bit easier to use it. And the total amount of code doesn't really grow.